fragments/tap-workload/accelerator.yaml

```
@@ -1,25 +1,27 @@
 engine:
-  condition: "#bsGitRepository != null"
-  merge:
-    - includes: [ "**/workload.yaml" ]
-      onConflict: NWayDiff
-      merge:
-        - type: ReplaceText # No-Op. In combination with NWayDiff, allows keeping the workload.yaml
-          # file untouched if the $.spec.source.git.url is NOT present (OpenRewrite emits nothing otherwise)
-        - type: OpenRewriteRecipe
-          recipe: org.openrewrite.yaml.ChangeValue
-          options:
-            oldKeyPath: '"$.spec.source.git.url"'
-            value: "#repoUrl.scheme + '://' + #repoUrl.host + '/' + #repoUrl.queryParams['owner'][0] + '/' + #repoUrl.queryParams['repo'][0]"
-        - type: OpenRewriteRecipe
-          recipe: org.openrewrite.yaml.ChangeValue
-          options:
-            oldKeyPath: '"$.spec.source.git.ref.branch"'
-            value: "#bsGitBranch"
-        - type: OpenRewriteRecipe
-          recipe: org.openrewrite.yaml.DeleteProperty
-          options:
-            propertyKey: '"$.spec.source.git.subPath"' # subPath is not supported atm
-      let:
-        - name: repoUrl
-          expression: "T(org.springframework.web.util.UriComponentsBuilder).fromUriString(#bsGitRepository).build(true)"
\ No newline at end of file
+  chain:
+    - condition: "#bsGitRepository != null"
+      chain:
+      - includes: [ "**/workload.yaml" ]
+        let:
+          - name: repoUrl
+            expression: "T(org.springframework.web.util.UriComponentsBuilder).fromUriString(#bsGitRepository).build(true)"
```

From discussion in slack:

```suggestion
            expression: "T(org.springframework.web.util.UriComponentsBuilder).fromUriString('https://' + #bsGitRepository).build(true)"
```

---

Done